crates/tbdex/src/json_schemas/json_schema_org/draft-07.json

```diff
@@ -0,0 +1,172 @@
+{
```

Comment:
so we'll need to make sure this is bundled _inside_ the byte code (dylib/so file) rather than being an adjacent file

that was the primary reason for stringifying the tbdex schemas inside `generated.rs` -- I know it feels a bit hacky, but seems pragmatic to me

btw we'll need a way to test this prior-to merging, to ensure an outbound request to http://json-schema.org is never made -- I can think of a few, but if you want to simply run a debug and step through the custom resolver to ensure that is the case that would be fine with me for the time being